open-metadata-implementation/access-services/subject-area/subject-area-server/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/server/services/SubjectAreaRelationshipRESTServices.java

```diff
@@ -1935,6 +1935,107 @@ public SubjectAreaRelationshipRESTServices() {
      */
     public SubjectAreaOMASAPIResponse<SemanticAssignment> getSemanticAssignmentRelationship(String serverName, String userId, String guid) {
         String restAPIName = "getSemanticAssignmentRelationship";
-        return restoreLine(serverName, restAPIName, userId, SemanticAssignmentMapper.class, guid);
+        return getLine(serverName, restAPIName, userId, SemanticAssignmentMapper.class, guid);
+    }
+
+    /**
+     * Update a CategoryHierarchyLink Relationship. A relationship between a Category and a Category.
+     *
+     * @param serverName           serverName under which this request is performed, this is used in multi tenanting to identify the tenant
+     * @param userId               userId under which the request is performed
+     * @param guid       guid of the CategoryAnchor relationship
+     * @param categoryHierarchyLink the categoryAnchor relationship
+     * @param isReplace            flag to indicate that this update is a replace. When not set only the supplied (non null) fields are updated.
+     * @return response, when successful contains the updated categoryHierarchyLink
+     * when not successful the following Exception responses can occur
+     * <ul>
+     * <li> UserNotAuthorizedException           the requesting user is not authorized to issue this request.</li>
+     * <li> InvalidParameterException            one of the parameters is null or invalid.</li>
+     * <li> PropertyServerException              Property server exception. </li>
+     * </ul>
+     */
+    public SubjectAreaOMASAPIResponse<CategoryHierarchyLink> updateCategoryHierarchyLink(String serverName, String userId, String guid, CategoryHierarchyLink categoryHierarchyLink, Boolean isReplace) {
+        String restAPIName = "updateCategoryHierarchyLink";
+        return updateLine(serverName,restAPIName, userId, guid, CategoryHierarchyLinkMapper.class, categoryHierarchyLink, isReplace);
+    }
+
+    /**
+     * Create a CategoryHierarchyLink Relationship. A relationship between a Category and a Category.
```

Comment:
It is more than a relationship between a category and another category, it is the relationship between a child category and its parent category. I think it is defined in the UML as "Relationship between two categories used to create nested categories."

 It is possible to create this relationship by specifying a category with a parentCategory